addons/account/models/account_tax.py

```diff
@@ -288,10 +288,10 @@ def _compute_amount(self, base_amount, price_unit, quantity=1.0, product=None, p
             return base_amount - (base_amount / (1 + self.amount / 100))
         # base / (1 - tax_amount) = new_base
         if self.amount_type == 'division' and not price_include:
-            return base_amount / (1 - self.amount / 100) - base_amount if (1 - self.amount / 100) else 0.0
+            return base_amount - (base_amount / (1 + self.amount / 100))
         # <=> new_base * (1 - tax_amount) = base
         if self.amount_type == 'division' and price_include:
-            return base_amount - (base_amount * (self.amount / 100))
+            return base_amount * self.amount / 100
```

Comment:
@smetl this use case is working properly now (the Brazilian need) (https://48557332-14-0-all.runbot118.odoo.com/web#id=74&action=448&model=account.move&view_type=form&cids=1&menu_id=579):
<img width="1203" alt="image" src="https://github.com/odoo/odoo/assets/45669815/562f3608-0e0f-4e0f-85f0-fc52ef9e8dfd">

cc' @tsb-odoo @chklop